eviatar/enable_windows_features.ps1

```diff
@@ -0,0 +1,33 @@
+﻿function Install-WindowsFeatures {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string[]]$Features
+    )
+    
+    foreach ($Feature in $Features) {
+        Write-Host "Installing feature: $Feature"

```

Comment:
What happens if the feature is mispelled or does not exist for some reason?

Reply:
If the feature name is misspelled or does not exist, the Install-WindowsFeature cmdlet will throw an error. To handle this, I can wrap the installation process in a Try-Catch block to capture the error and provide meaningful feedback.